src/modules/Auth/components/Login/FormLogin/container.js

```diff
@@ -30,27 +30,33 @@ import get from 'lodash/get';
 import FormLogin from './presenter';
 
 function mapStateToProps(state) {
-  const authMethod = get(state, 'auth.auth.method', authMethods.NATIVE);
+  const authMethod = get(state, 'auth.authMethod.data.result.userOrigin', authMethods.NATIVE);
   const isUnactivated = get(state, 'form.login.submitErrors.unactivated', false);
   const userEmail = get(state, 'form.login.values.username', '');
   const isLoading = get(state, 'auth.auth.isLoading', false);
+  const username = get(state, 'form.login.values.username', '');
```

Comment:
You will get these params as input variable on doSubmit call, so no need in such lines